optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -1152,6 +1157,42 @@ def to(self, device: Union[torch.device, str, int]):
         return self
 
 
+    @staticmethod
+    def infer_onnx_filename(
```

Comment:
it seems the only difference with the method in `modeling_ort` is the `return onnx_files[0].parent.name + "/" + onnx_files[0].name`, is the duplication really necessary?

Reply:
The point is to have `infer_onnx_filename()` return `encoder/encoder_model.onnx` instead of just `encoder_model.onnx`
So we can either separate the function's logic for each `modeling_ort` `modeling_decoder` and `modeling_seq2seq`  or we can add an argument to `infer_onnx_filename()` which lets us control that

Reply:
yes, I'd rather do a

`for name, folder name in zip([...], [...]):` with the three files than duplicate the method.